nucypher/blockchain/eth/sol/compile/solc.py

```diff
@@ -54,7 +54,7 @@ def __execute(compiler_version: VersionString, input_config: Dict, allow_paths:
     errors = compiler_output.get('errors')
     if errors:
         formatted = '\n'.join([error['formattedMessage'] for error in errors])
-        SOLC_LOGGER.warn(f"Errors during compilation: \n{formatted}")
```

Comment:
Just checking this is intentional, you mean to stop logging solc compile warnings?

Reply:
It was not, thank you for catching it

Reply:
✔️